lightning/get_invoice.js

```diff
@@ -25,36 +26,36 @@ const msPerSec = 1e3;
 */
 module.exports = ({id, lnd}, cbk) => {
   if (!id) {
-    return cbk([400, 'ExpectedId']);
+    return promise_helper([400, 'ExpectedId'], cbk);
```

Comment:
Why does `promise_helper` get passed `cbk` here but not in other error cases?

Reply:
It should be for all of the calls.  I put this together to start the talking points to make sure there were no issues if we take this approach with the abstraction (I think the `require` on line 2 is actually pointing to the wrong file so this wouldn't even run :))